alf/algorithms/rl_algorithm.py

```diff
@@ -402,7 +398,7 @@ def predict(self, time_step: ActionTimeStep, state, epsilon_greedy):
     PREPARE_SPEC = 3
 
     @abstractmethod
-    def rollout(self, time_step: ActionTimeStep, state, mode):
+    def rollout(self, time_step: TimeStep, state, mode):
```

Comment:
Maybe now `rollout` can be better called `predict_step_for_training`? Not to confuse it with `unroll`

Reply:
```rollout()``` -> ```rollout_step()```